updater/lib/dependabot/api_client.rb

```diff
@@ -170,18 +193,25 @@ def increment_metric(metric, tags:)
 
     private
 
-    attr_reader :base_url, :job_id, :job_token
+    sig { returns(String) }
+    attr_reader :base_url
+    sig { returns(T.any(String, Integer)) }
+    attr_reader :job_id
+    sig { returns(String) }
+    attr_reader :job_token
```

Comment:
Incredibly pedantic, but I find this quite hard to read, and think its a bit better with the spaces?

```suggestion
    sig { returns(String) }
    attr_reader :base_url

    sig { returns(T.any(String, Integer)) }
    attr_reader :job_id

    sig { returns(String) }
    attr_reader :job_token
```

Reply:
Works for me, can we make a rubocop rule that enforces this?